[2025-04-17T20:45:41.506+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-17T20:45:41.531+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_to_gcs.download_csv_from_kaggle scheduled__1987-01-01T00:00:00+00:00 [queued]>
[2025-04-17T20:45:41.539+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_to_gcs.download_csv_from_kaggle scheduled__1987-01-01T00:00:00+00:00 [queued]>
[2025-04-17T20:45:41.541+0000] {taskinstance.py:2867} INFO - Starting attempt 6 of 7
[2025-04-17T20:45:41.559+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_csv_from_kaggle> on 1987-01-01 00:00:00+00:00
[2025-04-17T20:45:41.568+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_to_gcs', 'download_csv_from_kaggle', 'scheduled__1987-01-01T00:00:00+00:00', '--job-id', '544', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_to_gcs.py', '--cfg-path', '/tmp/tmpvnsv4urr']
[2025-04-17T20:45:41.571+0000] {standard_task_runner.py:105} INFO - Job 544: Subtask download_csv_from_kaggle
[2025-04-17T20:45:41.573+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1013) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-17T20:45:41.574+0000] {standard_task_runner.py:72} INFO - Started process 1021 to run task
[2025-04-17T20:45:41.665+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_to_gcs.download_csv_from_kaggle scheduled__1987-01-01T00:00:00+00:00 [running]> on host f21ba0bbe878
[2025-04-17T20:45:41.763+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_to_gcs' AIRFLOW_CTX_TASK_ID='download_csv_from_kaggle' AIRFLOW_CTX_EXECUTION_DATE='1987-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1987-01-01T00:00:00+00:00'
[2025-04-17T20:45:41.765+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-17T20:45:41.785+0000] {logging_mixin.py:190} INFO - Dataset URL: https://www.kaggle.com/datasets/viggo123/data-fight-usa-dataset
[2025-04-17T20:45:58.687+0000] {logging_mixin.py:190} INFO - Downloading 1987.csv to /tmp/kaggle_downloads
[2025-04-17T20:45:58.694+0000] {logging_mixin.py:190} WARNING -   0%|          | 0.00/121M [00:00<?, ?B/s]
[2025-04-17T20:45:58.795+0000] {logging_mixin.py:190} WARNING -  77%|#######6  | 93.0M/121M [00:00<00:00, 973MB/s]
[2025-04-17T20:45:58.838+0000] {logging_mixin.py:190} WARNING - 100%|##########| 121M/121M [00:00<00:00, 888MB/s] 
[2025-04-17T20:45:58.841+0000] {python.py:240} INFO - Done. Returned value was: /tmp/kaggle_downloads/1987.csv
[2025-04-17T20:45:58.873+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-17T20:45:58.874+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=kaggle_to_gcs, task_id=download_csv_from_kaggle, run_id=scheduled__1987-01-01T00:00:00+00:00, execution_date=19870101T000000, start_date=20250417T204541, end_date=20250417T204558
[2025-04-17T20:45:58.940+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-17T20:45:58.975+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-17T20:45:58.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
